### DISCUSSION TITLE
Github Enterprise server integration with MS Teams

### DISCUSSION THREAD

**Discussion by shashikantvermaji** [community/community#38482] [Enterprise]
Hi All, 

I am trying to integrate my Github enterprise server with Teams but not even find out any related article even.
All  the documents available related to GITHUB.com(Clould) version.

As much as i know till now we need to set incoming Webhook and send request as REST API .

Kindly help me into this.

Regards,
Shashi

**Comment by david-engelmann** [comment]
[This post does a great job breaking down the steps you'll need to take](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/github-code-better-together-with-github-and-microsoft-teams/ba-p/659444). Please feel free to ask any questions that arise!

**Reply by ghostinhershell** [reply]
Hey @shashikantvermaji 👋

If @david-engelmann's response here answered your question, we'd appreciate you circling back and marking their response as the answer to this post. Otherwise, feel free to follow-up with any additional questions. Cheers!
